requirements.txt

```
@@ -9,5 +9,5 @@ torchvision>=0.7.0
 tqdm
 h5py
 structlog
-pytorch-lightning==0.8.5
+pytorch-lightning>=1.1.0
```

```suggestion
pytorch-lightning>=1.3.0
```
unless there is a reason we don't want to use the most recent version? 
#128